algebra/archimedean.lean

```diff
@@ -220,6 +220,15 @@ let ⟨q, h₁, h₂⟩ := exists_rat_btwn $
   lt_trans ((sub_lt_self_iff x).2 ε0) ((lt_add_iff_pos_left x).2 ε0) in
 ⟨q, abs_sub_lt_iff.2 ⟨sub_lt.1 h₁, sub_lt_iff_lt_add.2 h₂⟩⟩
 
+lemma pow_unbounded_of_gt_one {x : α} (y : α) : 1 < x → ∃ n : ℕ, y < monoid.pow x n := begin
+  assume x1,have x0 : 0 < x - 1 := by rwa [lt_sub_iff,zero_add],
+  cases exists_nat_gt (y / (x - 1)) with n hn,
+  have := pow_ge_one_add_mul (le_of_lt x0) n,
+  existsi n,rw [div_lt_iff x0,mul_comm] at hn,
```

Comment:
`rw ...` on a new line

Reply:
also why not merge with the next `rw`? are this two different subgoals? Use `{...}`

Reply:
I didn't merge them because they're at two different hypotheses

Reply:
Ah, sorry. I didn't see the `at ...` at the end